assets/src/components/propertiesPanel/ghostPropertiesPanel.tsx

```diff
@@ -6,12 +6,30 @@ import { Route } from "../../schedule"
 import PropertiesList from "../propertiesList"
 import BlockWaiverList from "./blockWaiverList"
 import Header from "./header"
+import IconAlertCircle from "../iconAlertCircle"
+import { isLateVehicleIndicator } from "../../models/ghost"
 
 interface Props {
   selectedGhost: Ghost
   route?: Route
 }
 
+const NoWaiverBanner = ({ ghost: { runId } }: { ghost: Ghost }) => (
+  <div className="m-ghost-properties-panel__no-waiver-banner">
+    <div className="m-ghost-properties-panel__no-waiver-banner-header">
+      <span className="m-ghost-properties-panel__no-waiver-banner-alert-icon">
+        <IconAlertCircle />
+      </span>
+      <div className="m-ghost-properties-panel__no-waiver-banner-title">
+        Unknown Ghost Bus - No Dispatcher Note
+      </div>
+    </div>
+    A ghost bus or dropped trip has been automatically detected on this route.
+    Please follow up with dispatch about Run {runId || "Not Available"} as
```

Comment:
This will read funny if the run is missing:
"Please follow up with dispatch about Run Not Available." Should we rephrase it in that case?

Reply:
Good question. What do you think, @ingridpierre?

Reply:
I spoke with Ingrid and we decided to remove the run number in all cases.